apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -606,7 +606,9 @@ def from_anndata(
 
         # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
         # MS/meas
```

Comment:
In line 598 above this we are setting `experiment_ms_uri` using `f"{experiment_uri}/ms"` instead of `_util.uri_joinpath`. Could that cause a problem for TileDB uris?

Reply:
Yeah I'll take a look at this again - I did originally initially try `sanitize_path(path)` at the beginning of the function, but it broke a few tests (since `path` is not always just a filename), so I just reverted it. Looking at the code again though, I think after it calls `urllib.parse.urlunparse(parts)`, I can just take the filename and sanitize that at the end.